river/command.zig

```diff
@@ -155,6 +158,8 @@ pub fn errToMsg(err: Error) [:0]const u8 {
         Error.InvalidOrientation => "invalid orientation. Must be 'horizontal', or 'vertical'",
         Error.InvalidRgba => "invalid color format, must be hexadecimal 0xRRGGBB or 0xRRGGBBAA",
         Error.InvalidValue => "invalid value",
+        Error.CannotReadFile => "cannot read file",
+        Error.CannotParseFile => "cannot parse file",
```

Comment:
```suggestion
```

Reply:
I think that providing a read error and a parse error has meaningful difference, for example, providing a wrong path leads to a read error, while providing a path to a file with contents that XKB cannot compile leads to a parse error.